developer/src/kmc-kmn/src/kmw-compiler/write-compiled-keyboard.ts

```diff
@@ -376,7 +376,10 @@ export function WriteCompiledKeyboard(
       else {
         result +=
           `${FTabStop+FTabStop}if(!m) {${nl}`+
-          `${FTabStop+FTabStop+FTabStop}${JavaScript_OutputString(keyboard, FTabStop + FTabStop + FTabStop, null, fgp.dpNoMatch, fgp)}${nl}`+
+          `${FTabStop+FTabStop}${JavaScript_OutputString(keyboard, FTabStop + FTabStop + FTabStop, null, fgp.dpNoMatch, fgp)}${nl}`+
+          // TODO: add FTabStop to line above, once we have 100% identical match
+          //       to legacy compiler, but because legacy compiler missed this
+          //       tabstop we'll leave it out here for now
```

Comment:
I may do this later by patching both this code and the legacy kmw compiler so that we can maintain byte-for-byte consistency until we are confident in the new compiler toolchain.